File: service/src/main/java/com/ooftf/service/widget/toolbar/TailoredToolbar.java
package com.ooftf.service.widget.toolbar;

import android.content.Context;
import android.util.AttributeSet;

import androidx.annotation.Nullable;
import androidx.core.content.ContextCompat;

import com.ooftf.master.widget.toolbar.official.ToolbarPlus;
import com.ooftf.service.R;

/**
 * @author master
 * @date 2017/10/10 0010
 */
public class TailoredToolbar extends ToolbarPlus {


    public TailoredToolbar(Context context) {
        super(context);

    }

    public TailoredToolbar(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public TailoredToolbar(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    protected int getDefaultBackgroundColor() {
        return  ContextCompat.getColor(getContext(),R.color.colorPrimary);
    }
}

File: service/src/main/java/tf/oof/com/service/base/TailoredToolbar.kt
package tf.oof.com.service.base

import android.app.Activity
import android.content.Context
import android.support.v7.widget.Toolbar
import android.util.AttributeSet
import android.util.Log
import android.view.Gravity
import android.widget.TextView
import tf.oof.com.service.R

/**
 * Created by master on 2017/10/10 0010.
 */
class TailoredToolbar :Toolbar {
    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs)
    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr)

    init {
        setNavigationOnClickListener { (context as Activity).finish() }
        setBackgroundColor(resources.getColor(R.color.colorPrimary))
    }

    override fun setTitle(title: CharSequence?) {
        super.setTitle(title)
    }
}